src/renderer/ui/components/generic/LyricsViewer.js

```diff
@@ -139,15 +139,17 @@ class LyricsViewer extends Component {
 
   render() {
     const barStyle = {};
+    // would rather not hardcode for default theme but not sure if it's passed through any props
+    // dynamically get from webpage?
+    barStyle.backgroundColor = 'rgb(252, 88, 37)';
```

Comment:
There are constants for these values in a LESS file somewhere, not sure if they are in JS anywhere.  Regardless I am happy with them being defined here for now 👍